content/en/agent/guide/heroku-ruby.md

```diff
@@ -625,10 +625,116 @@ Navigate to the [Service list][20] to see all your application services and your
 
 Next, enable logs. There are two options on how to send logs from your application to Datadog: setting up a Heroku log drain or using the Datadog Log Agent directly. Each of those have their benefits and limitations, but the good news is that you can set up both!
 
-The main disadvantage of the log drain is that it cannot correlate logs with traces, but this is possible using the Datadog Agent.
+The main disadvantage of the log drain is that all logs will arrive to Datadog from the same `ddsource`, usually `heroku`, so automatic parsing of logs using integrations (other than Heroku) won't happen.
 
 The main disadvantage of sending logs through the Datadog Agent is that Heroku system logs and router logs won’t be sent (you can send these using the log drain method).
 
+### Generating your Rails logs
+
+To configure your Rails logs, Datadog recommends using lograge. For this sample application we will set it so that logs and traces are correlated.
```

Comment:
```suggestion
To configure your Rails logs, Datadog recommends using lograge. For this sample application, set it so that logs and traces are correlated.
```